app/Services/Entity/RecoverySetupService.php

```diff
@@ -79,13 +79,14 @@ protected function elements(): array
             $element->date = \Carbon\Carbon::createFromTimeStamp(strtotime($element->deleted_at))->diffForHumans();
             $element->position = $key;
         }
+        //This will cast each object in the array to an array, and the toArray gets you from the collection back to an array.
```

Comment:
I'm now more confused as before. I can see what the code does, the comment needs to explain _why_ this monstrosity is needed